ConcernsCaseWork/ConcernsCaseWork.Service/Decision/DecisionDto.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ConcernsCaseWork.API.Contracts.Enums;
```

Comment:
I think you need to merge with the decisions feature as this dto seems similar to one we have created in the api contracts folder. Have a word with Mike and he'll show you. It should work (I think)